module/VuFind/src/VuFind/Db/Row/AccessTokenEntity.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Row Definition for access_token
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
+ * Copyright (C) The National Library of Finland 2022-2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Db_Row
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Db\Row;
+
+use VuFind\Db\Entity\AccessTokenEntityInterface;
+
+/**
+ * Row Definition for access_token
+ *
+ * @category VuFind
+ * @package  Db_Row
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class AccessTokenEntity extends RowGateway implements AccessTokenEntityInterface
```

Comment:
I would just call this AccessToken. While having "Entity" in the name has some advantages, I think it's better to remain consistent with our existing naming convention for row classes.